client/src/views/auth/Login.vue

```diff
@@ -7,30 +7,12 @@
         >
           <div class="rounded-t mb-0 px-6 py-6">
             <div class="text-center mb-3">
-              <h6 class="text-blueGray-500 text-sm font-bold">Sign in with</h6>
-            </div>
-            <div class="btn-wrapper text-center">
-              <button
-                class="bg-white active:bg-blueGray-50 text-blueGray-700 font-normal px-4 py-2 rounded outline-none focus:outline-none mr-2 mb-1 uppercase shadow hover:shadow-md inline-flex items-center font-bold text-xs ease-linear transition-all duration-150"
-                type="button"
-              >
-                <img alt="..." class="w-5 mr-1" :src="github" />
-                Github
-              </button>
-              <button
-                class="bg-white active:bg-blueGray-50 text-blueGray-700 font-normal px-4 py-2 rounded outline-none focus:outline-none mr-1 mb-1 uppercase shadow hover:shadow-md inline-flex items-center font-bold text-xs ease-linear transition-all duration-150"
-                type="button"
-              >
-                <img alt="..." class="w-5 mr-1" :src="google" />
-                Google
-              </button>
+              <h6 class="text-blueGray-500 text-sm font-bold">Sign in</h6>
```

Comment:
Issues with colours on login page
![image](https://github.com/tomstrong64/MyJourneyFootprint-COM617/assets/128226866/cc4a2fc5-a27f-4b85-b50c-c28e1183ddd6)

Unsure what colours register is supposed to be, but just for reference this is what it looks like for me
![image](https://github.com/tomstrong64/MyJourneyFootprint-COM617/assets/128226866/9cc335ba-d91f-42b7-817b-20ccc7bc4adc)